config/method.go

```diff
@@ -71,6 +72,8 @@ func parseMethodLine(loader *pkgload.PackageLoader, c *Converter, m *Method, val
 		var s string
 		s, err = parseString(rest)
 		m.AutoMap = append(m.AutoMap, strings.TrimSpace(s))
+	case "init":
```

Comment:
I think I like `default` better as the name for this feature. because you basically define the default value for types.

Reply:
yeah it makes totally sense, I am not that good at naming :)